Bottom: 5debe9fe8e9952bd520df07b40c18851ff0981c6
Top:    9e06ada3d04d8b8e361e5e036839e51c95183196
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-08 20:07:08 +0100

Refresh of work-in-progress-on-subcomposi

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 9b1bc3a..3c12833 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -80,8 +80,6 @@ ghid_set_layer (const char *name, int group, int empty)
       idx = PCB->LayerGroups.Entries[group][idx];
     }
 
-#define SUBCOMPOSITE_LAYERS
-#ifdef SUBCOMPOSITE_LAYERS
   /* Flush out any existing geoemtry to be rendered */
   hidgl_flush_triangles (&buffer);
 
@@ -93,7 +91,6 @@ ghid_set_layer (const char *name, int group, int empty)
   glStencilFunc (GL_GREATER, stencil_bit, stencil_bit);       /* Pass stencil test if our assigned bit is clear */
   glStencilMask (stencil_bit);                                /* Only write to our subcompositing stencil bitplane */
   priv->subcomposite_stencil_bit = stencil_bit;
-#endif
 
   if (idx >= 0 && idx < max_copper_layer + 2)
     {
